product.py

```diff
@@ -566,6 +566,19 @@ def findXenSourceBackups():
             backup = XenServerBackup(p, b.mount_point)
             logger.log("Found a backup: %s" % (repr(backup),))
 
+            # Don't restore a BIOS backup with a UEFI installer and conversely
+            backup_is_uefi = False
```

Comment:
nitpicking and possibly a matter of taste: that one would fit neatly in an `else` arm of the `for` loop

Reply:
TIL `for: else:` loop.
Fixed, thanks